lib/dradis/plugins/burp/importer.rb

```diff
@@ -63,12 +70,21 @@ def import(params = {})
 
         evidence_text = template_service.process_template(
           template: 'evidence',
-          data: xml_issue)
+          data: xml_issue
+        )
+
+        if evidence_text.include?(::Burp::Issue::INVALID_UTF_REPLACE)
```

Comment:
2 questions:
We are changing evidence text based on an issue constant?
Should the constant be `::Burp::INVALID_UTF_REPLACE` instead?

Where are we really replacing the evidence text?

Reply:
@dormi 
- I've updated the namespacing to improve the semantics.
- We are replacing the evidence text at the Issue class (note that this Issue refers to the Burp Issue not Dradis).

Reply:
:+1: